src/gui/editors/PianoRoll.cpp

```diff
@@ -374,6 +374,23 @@ PianoRoll::PianoRoll() :
 	connect( &m_noteLenModel, SIGNAL( dataChanged() ),
 					this, SLOT( noteLengthChanged() ) );
 
+	// Set up key selection
+	m_keyModel.addItem(tr("No key"));
+	m_keyModel.addItem(tr("C"));
+	m_keyModel.addItem(tr("C♯/D♭"));
+	m_keyModel.addItem(tr("D"));
+	m_keyModel.addItem(tr("D♯/E♭"));
+	m_keyModel.addItem(tr("E"));
+	m_keyModel.addItem(tr("F"));
+	m_keyModel.addItem(tr("F♯/G♭"));
+	m_keyModel.addItem(tr("G"));
+	m_keyModel.addItem(tr("G♯/A♭"));
+	m_keyModel.addItem(tr("A"));
+	m_keyModel.addItem(tr("A♯/B♭"));
+	m_keyModel.addItem(tr("B"));
```

Comment:
@Spekular so what if I used the same note strings as the piano keys?
```suggestion
	for (int i = 0; i < 12; ++i) { m_keyModel.addItem(tr(s_noteStrings[i].toStdString().data())); }
```

Reply:
Clever!